### DISCUSSION TITLE
Teleport database access to digital ocean managed db

### DISCUSSION THREAD

**Discussion by siexp** [gravitational/teleport#7531] [Q&A] [answered]
Hi guys,

I'm trying to configure teleport db access to digital ocean managed MySql 8 DB but encountered with a problem that teleport requires configuring ssl certificate on the mysql server side however Digital Ocean do not provide such feature.

When I trying to configure without ssl cert I got either problem with SSL connection with client side certs or problem with connection to teleport when no client certs provided.

Have anyone an idea of some workaround here? Maybe there is a way to disable SSL validation for db access?

**Comment by webvictim** [comment, marked as answer]
As far as I know, there's no way to connect to a MySQL database without using client certificate authentication - Teleport does not use usernames/passwords for database authentication, only client certificates.

cc @r0mant

**Reply by siexp** [reply]
So the only possible workaround is to create user with only permission to open SSH tunnel from droplet to the DB?

**Reply by webvictim** [reply]
That sounds like one possible workaround, for sure.
